akka-docs/rst/java/lambda-actors.rst

```diff
@@ -1,8 +1,8 @@
-.. _lambda-actors-java:
+.. _actors-java:
 
```

Comment:
Before merging I will also rename those doc files (remove the lambda prefix). Kept it now to not mess up the diff.